src/main/java/nextstep/subway/domain/Distance.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.subway.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class Distance {
+    @Column(name = "distance")
+    private Long distance;
+
+    protected Distance() {
```

Comment:
```suggestion
    protected Distance() {}
```
이 부분은 [컨벤션](https://naver.github.io/hackday-conventions-java/#permit-concise-empty-block)에 따르면 이렇게 써도 될 것 같아요...!

Reply:
오 감사합니다.